htpasswd_cr.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: config.openshift.io/v1
+kind: OAuth
+metadata:
+  name: cluster
+spec:
+  identityProviders:
+  - name: htpasswd_provider 
+    mappingMethod: claim 
+    type: HTPasswd
+    htpasswd:
+      fileData:
+        name: htpass-secret 
```

Comment:
missimg something?

Reply:
Nope, its done!
Wait did you find something?

Reply:
It says: `filedata`, but i do not see a file ... ?

Reply:
So the htpasswd file content is stored as a secret in OpenShift which has the name `htpass-secret`, also instead of adding a new file (with just one line), the file's content is stored in a variable, both these changes are in the `createdisk.sh` file.